src/main/java/pages/GuestCheckoutPage.java

```diff
@@ -0,0 +1,84 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import static engine.drivers.WebDriverFactory.getWebDriverInstance;
+
+public class GuestCheckoutPage {
+    protected BasePage basePage;
+
+    // Locators:
+    public By firstnameInputLocator = By.id("AccountFrm_firstname");
+    public By lastnameInputLocator = By.id("AccountFrm_lastname");
+    public By emailInputLocator = By.id("AccountFrm_email");
+    public By telephoneInputLocator = By.id("AccountFrm_telephone");
+    public By faxInputLocator = By.id("AccountFrm_fax");
+    public By companyInputLocator = By.id("AccountFrm_company");
+    public By address1InputLocator = By.id("AccountFrm_address_1");
+    public By address2InputLocator = By.id("AccountFrm_address_2");
+    public By cityInputLocator = By.id("AccountFrm_city");
+    public By zoneIdInputLocator = By.xpath("//ul[contains(@class, 'currency')]/parent::*");
+    public By postcodeInputLocator = By.id("AccountFrm_postcode");
+    public By countryIdInputLocator = By.xpath("//ul[contains(@class, 'AccountFrm_country_id')]/parent::*");
+    public By loginNameInputLocator = By.id("AccountFrm_loginname");
+    public By passwordInputLocator = By.id("AccountFrm_password");
+    public By passConfirmInputLocator = By.id("AccountFrm_confirm");
+    public By newsletterInputLocator = By.xpath("//*[@id='AccountFrm_newsletter1']/button");
+
+
+    public GuestCheckoutPage() {
+        this.basePage = new BasePage();
+    }
+
+    public void enterFirstName(String firstname) { basePage.enterTextToElement(firstnameInputLocator, firstname); }
```

Comment:
Use auto-formatting on all of the files:)

Reply:
https://resources.jetbrains.com/storage/products/intellij-idea/docs/IntelliJIDEA_ReferenceCard.pdf
Reformat code & Optimize imports shortcuts are super useful!

Reply:
done